h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstTopN.java

```diff
@@ -0,0 +1,167 @@
+package water.rapids.ast.prims.reducers;
+
+import water.MRTask;
+import water.fvec.C8Chunk;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+import water.fvec.Vec;
+import water.rapids.Env;
+import water.rapids.ast.AstPrimitive;
+import water.rapids.ast.AstRoot;
+import water.rapids.vals.ValFrame;
+
+import java.util.PriorityQueue;
+
+import static java.lang.StrictMath.min;
+
+public class AstTopN extends AstPrimitive {
+  @Override
+  public String[] args() {
+    return new String[]{"frame", "col", "nPercent", "getBottomN"};
```

Comment:
Okay, I found both cases in use.  Will leave it as it is now until you guys make a decision later.